languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -34110,6 +34124,79 @@ though, and has since been slightly modified)-->
+             <message>Une virgule semble être requise dans ce contexte.</message>
+             <suggestion>\1 \2, \3 \4</suggestion>
+             <example correction="Nous montons, nous descendons"><marker>Nous montons nous descendons</marker> nous ne savons plus où aller.</example>
+         </rule>
+         <rule>
+             <antipattern>
+                 <token postag="R pers suj.*" postag_regexp="yes"/>
+                 <token inflected="yes">prendre</token>
+                 <token postag="V.*" postag_regexp="yes"/>
+                 <token>et</token>
+                 <token postag="V.*" postag_regexp="yes"/>
+             </antipattern>
+             <pattern>
+                 <token postag="R pers suj.*" postag_regexp="yes"/>
+                 <unify>
+                     <feature id="person"/>
+                     <feature id="tense"/>
+                     <marker>
+                         <token postag="V.*" postag_regexp="yes">
+                             <exception postag="V etre.*" postag_regexp="yes"/>
+                             <exception postag="V avoir.*" postag_regexp="yes"/>
+                         </token>
+                         <token postag="V.*" postag_regexp="yes" regexp="yes">(?-i)[a-z].*<exception regexp="yes">casques?|confiance</exception></token>
+                     </marker>
+                     <unify-ignore>
+                         <token>et</token>
+                     </unify-ignore>
+                     <token postag="V.*" postag_regexp="yes"/>
+                 </unify>
+             </pattern>
+             <message>Une virgule semble être requise dans ce contexte.</message>
+             <suggestion>\2, \3</suggestion>
+             <example correction="monte, descend">Elle <marker>monte descend</marker> et recule.</example>
+             <example>Elle <marker>prend place</marker> et trouve une feuille.</example>
+             <example>Il <marker>aime Sue</marker> et accepte tout pour elle.</example>
+             <example>Il <marker>reprend confiance</marker> et retourne au combat.</example>
+         </rule>
+         <rule>
+             <pattern>
+                 <token postag="R pers suj.*" postag_regexp="yes"/>
+                 <unify>
+                     <feature id="person"/>
+                     <feature id="tense"/>
+                     <marker>
+                         <token postag="V.*" postag_regexp="yes"><exception postag="V etre.*" postag_regexp="yes"/></token>
+                         <token postag="V.*" postag_regexp="yes"/>
+                         <token postag="V.*" postag_regexp="yes"/>
+                     </marker>
+                 </unify>
+             </pattern>
+             <message>Une virgule semble être requise dans ce contexte.</message>
+             <suggestion>\2, \3, \4</suggestion>
+             <example correction="monte, descend, recule">Elle <marker>monte descend recule</marker>.</example>
+         </rule>
```

Comment:
So, I put this rule, too, through the SentenceSourceChecker. I will send you the results, there are pitfalls like:

> Nous faisons table rase de votre passé de criminel.

"rase" is a `V.*`, but it is also tagged as `J f s` in this case.

Reply:
I modified the rule so it should be ready now.

Reply:
Can you test it with the SSC, please? Please let me know if the results are okay.

Reply:
I tested it using the SSC, the results seem good.